### DISCUSSION TITLE
Import Results from CKL(B) or XCCDF files Rejects Results

### DISCUSSION THREAD

**Discussion by Vesfoldy** [NUWCDIVNPT/stig-manager#1242] [Q&A]
I am having trouble importing a completed checklist file for the Application_Server_SRG STIG Benchmark.  STIG Manager is rejecting all of the results.  Any ideas what may be the problem or where I could look for more details?  See attached screenshot for details. 
![Untitled](https://github.com/NUWCDIVNPT/stig-manager/assets/153039856/0a4c2651-f418-460d-ac01-6ddeaf0ed673)

**Comment by cd-rite** [comment]
Hi @Vesfoldy The most common cause of this is when the checklist is based on version of the STIG that STIGMan does not know about.  You'll want to make sure you've imported the version of the STIG the checklist was based on via the Application Management-> STIG Benchmarks interface:
https://stig-manager.readthedocs.io/en/latest/admin-guide/admin-quickstart.html#stig-updates

**Reply by Vesfoldy** [reply]
I checked the version of the checklist (Ver 3, Rel 3) and the STIG Benchmark (Ver 3, Rel 4) I have installed.  I thought STIG Manager would process it if is was close enough.  Anyway, I am cannot find the Application Server SRG - Ver 3, Rel 3 STIG anywhere.  What do you suggest I do?  I am hoping manually transferring the content to the newer checklist is not my only option.

**Comment by cd-rite** [comment]
@Vesfoldy  STIGMan will generally roll with different versions, IF it knows about the specific Rule IDs in the STIGs. But in that case, DISA changed EVERY ruleID (And almost nothing else!!!):
<img width="658" alt="image" src="https://github.com/NUWCDIVNPT/stig-manager/assets/61710958/945e6344-e97a-4926-8ee4-693338642079">

Sadly, DISA doesn't offer their old versions of STIGs, BUT, I happened to have an old library that had the version you are looking for. Feel free to use it if you are comfortable with that. Once you import it as a reference STIG in App Management, the checklist should import without issue.
[U_Application_Server_V3R3_SRG.zip](https://github.com/NUWCDIVNPT/stig-manager/files/14619747/U_Application_Server_V3R3_SRG.zip)

**Reply by Vesfoldy** [reply]
I am comfortable with using it.  Loading it solved my issue.  Thanks!  
Would you happen to have these in your stash; ESS_ePO_5-3_5-9_STIG V2R12 STIG, ESXi_5.0_VM_STIG V1R7 STIG, vROps 8.x STIG and Remove_Storage_amd_External_Connnection_Technologies V1R7 STIG?  I am struggling to find those too.

**Reply by cd-rite** [reply]
Great! Sorry, I don't see those others!

**Reply by cd-rite** [reply]
@Vesfoldy I found the Remove Storage one here: https://www.stigviewer.com/stig/removable_storage_and_external_connections/
I downloaded the XML and it imported into STIGMan fine.  They might have some of the other ones you are looking for.

**Reply by Vesfoldy** [reply]
@cd-rite, I was able to get the ESXi_5.0_VM_STIG V1R7 STIG and Remove_Storage_amd_External_Connnection_Technologies V1R7 STIG from that site.  I will continue to look for the others.  Thanks for your help on this.
